extensions/mssql/src/tableDesigner/tableDesigner.ts

```diff
@@ -39,7 +42,8 @@ export function registerTableDesignerCommands(appContext: AppContext) {
 			schema: schema,
 			id: `${sqlProviderName}|${server}|${database}|${schema}|${name}`,
 			connectionString: connectionString,
-			accessToken: context.connectionProfile.options.azureAccountToken
+			accessToken: context.connectionProfile.options.azureAccountToken,
+			tableSubType: tableSubType
```

Comment:
same, we don't need this.

Reply:
table info needs the sub type and this is what the PR is about - https://github.com/microsoft/azuredatastudio/pull/18808/files#diff-644b218f64c3dde63820b8da0b6d2016845d482b4a308ac8186ae3519c899111R46

Reply:
my bad, yes, we do.